game/scripts/vscripts/components/duels/duels.lua

```diff
@@ -444,6 +445,23 @@ function Duels:ActuallyStartDuel(options)
     end
   end
 
+  --duel start invulnerability
+  --Invulnerability Duration/ Should probably move this to options
+  local invulnerabilityduration = 2.0
```

Comment:
This is bad naming, please use either camelCase or snake_case. Or at least something other than all lowercase with no separators.

Reply:
I would actually suggest going the opposite direction and making it a global set in `settings.lua`.